### ISSUE TITLE
Use different styles for member nodes vs snapped nodes

### ISSUE THREAD

**Issue by geohacker** [developmentseed/observe#271]
Currently we have identical styles for member nodes and new nodes that we are adding + snapping to. 
<img width="355" alt="Screenshot 2020-05-28 21 20 55" src="https://user-images.githubusercontent.com/371666/83164657-1a4a5980-a12a-11ea-9bb3-02d6686bfdbc.png">

We should have different styles so users can easily snap to existing nodes.

cc @LanesGood

**Comment by LanesGood** [comment]
@geohacker. we currently have just one style for `NearestNode` and a separate style for `EditingNodes`. When hovering over an existing way, only the `NearestNode` style is used. I do see the id Observe when logging the `NearestFeatures` feature collection, so I assume something like the below should be possible. These two approaches aren't working - any suggestions? 

```
const nearestNodes = {
  circleColor: { id === observe ? colors.primary : 'white'},
  ...
}
```

```
circleColor: [
    'match',
    ['get', 'id'],
    'observe/*', colors.primary,
    'white'
  ],
...
```

cc @sethvincent

**Comment by geohacker** [comment]
@LanesGood yes, though Mapbox GL doesn't support those wildcard matching. So I just added a new property called `membership: no` to distinguish these new nodes. Pushed to the PR and I think it's ready. Thank you!

**Comment by LanesGood** [comment]
Closed by #272
